src/ImageItem.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ImageItem from './ImageItem.js';
+import renderer from 'react-test-renderer';
+
+
```

Comment:
For these tests that require data, you could have also just passed an empty array, or pass your data file to the component you are testing. Makes it a little cleaner looking in your test files!